lib/parsec/fileparse.py

```diff
@@ -71,8 +71,9 @@
     ''',
     re.VERBOSE)
 
+# Designed to match lines ending '\ ' without matching '\   comment'
 _BAD_CONTINUATION_TRAILING_WHITESPACE = re.compile(
-    r'''.*\\\s''')
+    r'''/^([^#\n]+)?\\\s+$/gm''', re.VERBOSE)
```

Comment:
Ach, sorry @wxtim, I copied the regex in a kinda "generic" format where you have `/<the regex>/<modifiers>`, to execute this using Python's `re` module:

```python
import re
re.compile(r'^([^#\n]+)?\\\s+$', re.M)  # the g flag comes down to the method used (find|search|match...)
```

Reply:
I had rather assumed that the REGEX101 site had produced something sensible.